lib/main.coffee

```diff
@@ -68,6 +67,8 @@ module.exports =
   findExecutable: ->
     {spawnSync} = require 'child_process'
     consistentEnv = require 'consistent-env'
+    path = require 'path'
```

Comment:
Instead of `require`ing it every time, leave `path` in the global and just check if it's been required already before `require`ing it.

```coffeescript
path = null
# ...
findExecutable: ->
  path = require 'path' if not path?
# ...
provideLinter: ->
  path = require 'path' if not path?
```